src/main/steps/screen-questions/language-preference/content.ts

```diff
@@ -13,18 +20,25 @@ const cy: typeof en = {
   line2: 'Dewisiwch pa iaith yr hoffech eu cael.',
   yes: 'Cymraeg a Saesneg',
   no: 'Saesneg yn unig',
+  errors: {
+    languagePreferenceWelsh: {
+      required: 'Dewisiwch yr iaith yr hoffech gael eich dogfennau a’ch gohebiaeth.'
+    }
+  }
 };
 
 export const languagePreferenceForm = {
-  fields: {
-    languagePreferenceWelsh: {
+  fields: [
+    { name: 'languagePreferenceWelsh',
```

Comment:
I would prefer to use the key name, any reason you didnt?

Reply:
maybe its coz im rusty but i couldnt do any filtering or map functions to property, only thing could think of was to covert it into an array object but if you know any othe way im game.

Reply:
```
Object.values(form.fields).map(...)
```